nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/header/components/nf-ng-group-component.js

```diff
@@ -304,7 +461,7 @@
                     });
                 }).promise();
             }
-        }
+        };
```

Comment:
Is there a reason for adding this semicolon?

Reply:
It's not necessary to have the semicolon, but I thought it would be more complete.